cobalt/debug/console/content/debugger_client.js

```diff
@@ -16,6 +16,7 @@ function DebuggerClient() {
   this.DEBUGGER_DETACHED = 0;
   this.DEBUGGER_ATTACHING = 1;
   this.DEBUGGER_ATTACHED = 2;
+  this.DEBUGGER_DISABLED = 3;
```

Comment:
Do we really need this new state? Can't we just use ```this.DEBUGGER_DETACHED```

Reply:
Yes, because we also don't even want to try attaching in `DebuggerClient.prototype.attach` (right below here).